File: 004fibo/Scala/src/fibo.scala
object fibo extends App {

  var x1=1
  var x2=1
  var temp=0
  var n=10

  println(x1+"\n"+x2)

  for(i<-0 to n){
    temp = x2
    x2 += x1
    System.out.println(x2)
    x1 = temp
  }

  System.out.println("scalattenani")

  val lst = List(5,235,47,3,5346,355623);

  println(lst.max);
  println(lst.min);

}

File: 004fibo/Java/src/fibo.java

public class fibo {
    public static void main(String[] args){
        long x1=1;
        long x2=1;
        long temp;
        int n=10;

        System.out.println(x1+"\n"+x2);

        for(int i=0;i<n;i++){
            temp=x2;
            x2+=x1;
            System.out.println(x2);
            x1=temp;
        }
    }
}
